resources/js/web3_init.js

```diff
@@ -43,6 +43,25 @@ WebViewBridge.onMessage = function (message) {
             }
         }
     }
+
+    else if (data.type === "scan-qr-code-callback")
```

Comment:
couldn't we do this on clojurescript side instead of having to add code for each new event ?